### ISSUE TITLE
Description not showing in AR/AP Transaction > Print

### ISSUE THREAD

**Issue by aungzwin10** [ledgersmb/LedgerSMB#6482]
### Version

1.9.12

### What browsers are you seeing the problem on?

This problem isn't browser related

### What happened?

Step to reproduce

1. Create AR/AP > Add Transaction.
2. Add one or two income/expense line and write line description.
3. Click Save
4. Click Print. 

<img width="1196" alt="Screen Shot 2022-03-28 at 3 58 48 PM" src="https://user-images.githubusercontent.com/31498748/160369312-fbdd43d7-038f-4909-9476-5ee8321d0234.png">

<img width="1365" alt="Screen Shot 2022-03-28 at 3 53 54 PM" src="https://user-images.githubusercontent.com/31498748/160369499-e7e9fda9-f8f5-467f-b35d-a2255d531059.png">

The line description("general purchase description" and "medicine purchase description" in my example) are not showing in the print.
This happened for all templates `HTML, CSV, Postscript, PDF`.

### What should have happened?

According to the template code below, description should show after the amount.
https://github.com/ledgersmb/LedgerSMB/blob/a66d8a0bff573dfd1890e9ecb8cb3ea3e025d742/templates/demo/ap_transaction.html#L92-L101

<img width="1365" alt="Screen Shot 2022-03-28 at 3 53 54 PM" src="https://user-images.githubusercontent.com/31498748/160371066-96be2faf-ce53-4767-b3fa-cc50a3cb4127.png">

**Comment by ehuelsmann** [comment]
On my 1.9.12 installation (with templates inherited from waaayyyy back when, adjusted only when I run into trouble), I'm seeing this issue *only* on the AP side of things (which is expected, as that's the side you're using to showcase the problem.
The difference in rendering baffles me, to be frank, because there's very little difference in the code paths for the AR and AP transactions. Or at least, that's what I interpret from the code.

In other words: I need more time to figure this one out...

**Comment by ehuelsmann** [comment]
Ok. I found more time and it's ready for you to test. Please see the PRs mentioned above.

**Comment by ehuelsmann** [comment]
@aungzwin10 please also review https://ledgersmb.org/content/110-release-notes which describes which actions you should take to make a future-proof upgrade.
